### DISCUSSION TITLE
Is it possible to train with CPU only ?

### DISCUSSION THREAD

**Discussion by DreamError404** [bmaltais/kohya_ss#1632] [General]
我看到了一个用M1 MacBook训练lora的视频，所以我想尝试一下不用独显，仅使用cpu来训练，不知道有没有可能？（给核显分配的内存充足情况下）

I watched a video of training with a M1 Macbook，it's cool. I wonder if it is possible to train with a CPU like AMD 7840HS or some others?

**Comment by madrooky** [comment]
If it is still not possible to use AMD GPU's it is most likely not possible to AMD APU's as well. Same principle, no driver support.
